pir/cpp/payload.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package pir;
+
+message Payload {
+  repeated bytes query = 1;
+}
+
+message SessionPayload {
+  required Payload query = 1;
+  required int64 id = 2;
```

Comment:
Avoid required fields. Also: ID of what?

Reply:
ID is session ID